tests/test_wrappers.py

```diff
@@ -161,8 +164,9 @@ def test_obs_cast(dtype: np.dtype, episode_length: int = 5):
     assert obs == 0
```

Comment:
`env.reset()` should return `obs, info`, so both the env and this test need an update